packages/types/src/modules-sdk/index.ts

```diff
@@ -201,6 +201,18 @@ export interface ModuleServiceInitializeOptions {
     database?: string
     driverOptions?: Record<string, unknown>
     debug?: boolean
+    pool?: {
```

Comment:
same as other comment

Reply:
Something tells me it is wrong to add all these knex-specific options to the module config type.

Reply:
Yes for sure, maybe we should just have a Record type and reference the knex api doc for us?

Reply:
Yeah, that could make sense